include/taskolib/execute_lua_script.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \file   execute_lua_script.h
```

Comment:
I guess there is no code change here, just moving around?

I like the name change of the function :+1:

Reply:
Yes, no code change, just a renaming.